src/lib/number.ts

```diff
@@ -0,0 +1,14 @@
+import { millify } from 'millify';
+
+export function formatNumber(num: number | string): string {
```

Comment:
i have a few comments about this function. rather than comment on each line, i'll just throw them into one comment. haha.
- 'formatNumber' feels overly generic for something that's actually very specific. all we're actually using this for is formatting a token value, i think?
- i think we should be able to assume string input here.
- and. .. let's not cast to float. let's just keep things as a string.

Reply:
Hmm damn my comment disappeared, but I agree with these and have fixed them ☺️